src/VisualStudio/TestUtilities/Window/InteractiveWindow.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using Roslyn.VisualStudio.Test.Utilities.Remoting;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public class InteractiveWindow
+    {
+        private const string DteCSharpViewCommand = "View.C#Interactive";
+        private const string DteCSharpWindowTitle = "C# Interactive";
+        private const string DteReplResetCommand = "InteractiveConsole.Reset";
+
+        private const string ReplPromptText = "> ";
```

Comment:
If possible use the constants defined in the product code (although I'm not sure where they are.)